pyccel/stdlib/ndarrays/ndarrays.c

```diff
@@ -238,8 +238,8 @@ t_slice new_slice(int32_t start, int32_t end, int32_t step)
     slice.step = step;
     return (slice);
 }
-
-t_ndarray array_slicing(t_ndarray arr, int n, ...)
+#include <stdio.h>
+t_ndarray array_slicing(t_ndarray arr, int n, int n_slices, ...)

```

Comment:
I do not understand why we need both `n` and `n_slices`.

I think that a bit of documentation is really needed here. Could you please add a comment block using Doxygen style?

Reply:
The only reason why I added `n_slices` is because there was a test written like this where there 1 as number of slices but two slices were given:
`b = array_slicing(a, 1, new_slice(1, 2, 1), new_slice(0, a.shape[1], 1));`

Reply:
`n` is not the number of slices. I believe it is the number of dimensions in the view

Reply:
Could it be possible the number of slices be bigger than the size of `arr`?

Reply:
As there is no doc you will have to check that in the c printer or speak with the person that developed it (git blame says @nhamidn).

My understanding is that the number of slices is always equal to the number of dimensions in arr. Which may be more than `n` the number of dimensions in `view`